pure.zsh

```diff
@@ -593,7 +593,7 @@ prompt_pure_state_setup() {
 	[[ $UID -eq 0 ]] && username='%F{$prompt_pure_colors[user:root]}%n%f'"$hostname"
 
 	typeset -gA prompt_pure_state
-	prompt_pure_state[version]="1.10.3"
+	prompt_pure_state[version]="1.11.0"
```

Comment:
Maybe we can use https://github.com/sindresorhus/replace-in-files-cli to bump this when package.json is bumped.

Example: https://github.com/sindresorhus/modern-normalize/commit/c1d65e3f7daba2b695ccf837d2aef19d586d1ca6#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R19

Reply:
I had something similar in mind, that looks perfect!